.github/workflows/update-docs.yml

```diff
@@ -41,15 +39,14 @@ jobs:
 
       - name: Commit changes to docs/ folder (if any)
         run: |
-          git add . && git commit -m "Update docs/ folder" && git push origin ${{ env.BRANCH }} && echo "CREATE_PR=true" >> $GITHUB_ENV
+          if git add . && git commit -m "Update docs/ folder"; then git push origin ${{ env.BRANCH }} && echo "CREATE_PR=true" >> $GITHUB_ENV; else echo "CREATE_PR=false" >> $GITHUB_ENV; fi
 
       - name: Create PR
-        if: ${{ env.CREATE_PR }}
         env:

```

Comment:
curious to know why we can't use the `if` condition here so that we can avoid downloading the `hub` tool.

Reply:
Just tried a string comparison and it works as expected now